src/patternfly/components/OptionsMenu/examples/options-menu-single-example.hbs

```diff
@@ -0,0 +1,19 @@
+{{#> options-menu id="options-menu-single-example" options-menu--label-text="Choose number of items per page"}}
```

Comment:
Can we change this to something like "Page size" or "Items per page"? I like to think of these labels the same way we think about labels in a form, and we would use a label like "Choose number of..." for a select or "Enter your..." for a text input. The semantics provided by the aria attributes on the toggle communicate to screen reader users that they will be presented with choices when they act on the toggle.